master/getting-started/bare-metal/bare-metal.md

```diff
@@ -318,11 +318,11 @@ cat << EOF | calicoctl create -f -
   kind: HostEndpoint
   metadata:
     name: <name of endpoint>
-    node: <node name or hostname>
     labels:
```

Comment:
I don't see `labels` anymore in the struct. Did you try that out?

Reply:
works fine w/sample yaml and can be queried...
```
./calicoctl get hostEndpoint --out=go-template="{{range .}}{{range .Items}}{{.ObjectMeta.Labels}},{{end}}{{end}}"
map[role:webserver environment:production],map[host-endpoint:ingress],map[type:production],
```

Reply:
never mind, that's in `metav1.ObjectMeta`